api/src/main/java/io/strimzi/api/kafka/model/template/JmxTransQueryTemplate.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.template;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonPropertyOrder({
+        "targetMBean", "attributes", "outputs"})
+@EqualsAndHashCode
+public class JmxTransQueryTemplate implements Serializable, UnknownPropertyPreserving {
+    private String targetMBean;
+    private List<String> attributes;
+    private List<String> outputs;
+
+    private static final long serialVersionUID = 1L;
+    private Map<String, Object> additionalProperties = new HashMap<>(0);
+
+    @JsonProperty(required = true)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Description("If using a wildcard, the MBean(s) that the data is gathered from.")
+    public String getTargetMBean() {
+        return targetMBean;
+    }
+
+    public void setTargetMBean(String targetMBean) {
+        this.targetMBean = targetMBean;
+    }
+
+    @JsonProperty(required = true)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Description("Determine which attributes of the targeted MBean are read")
+    public List<String> getAttributes() {
+        return attributes;
+    }
+
+    public void setAttributes(List<String> attributes) {
+        this.attributes = attributes;
+    }
+
+    @JsonProperty(required = true)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @Description("List of names of output definitions, which specify where JMX metrics are pushed to, and in which data format")
```

Comment:
So the elements of this list are names _and data formats_? That sound like there's some kind of syntax involved. Can you give an example?